scripts/upgrade

```diff
@@ -156,6 +156,11 @@ ynh_print_info --message="Updating php dependencies..."
 
 chown -R "$app": "$final_path"
 
+# Bypass composer issue and GitHub Rate Limit https://developer.github.com/v3/#rate-limiting
+if [ ${PACKAGE_CHECK_EXEC:-0} -eq 1 ]; then
+	sleep 3600
```

Comment:
Uuuuuuuuh, 3600, really ? o.O 
That's like ... a one hour dead time on the CI each time this upgrade script is ran ? x_X

Reply:
I'm making test but there is an issue with composer and github Rate Limit during package check:

During that [package_check](https://ci-apps-dev.yunohost.org/jenkins/view/yalh76/job/drupal_ynh%20PR21%20(yalh76)/8/console) I didn't understand why :
```
Upgrade...
Start the LXC container.
Create witness files...
Working time: 10 seconds.
!!! Timeout reached (180 min). !!!
!!! Package check was too long, its execution was aborted. !!! (PCHECK_AVORTED)
> Arrêt de package_check
> Arrêt du conteneur
Arrêt du conteneur pchecker_lxc
VORTED)
```

So I tested manually: installing drupal and upgrading it. During upgrade I had:

```
120037 DEBUG + pushd /var/www/drupal
120038 DEBUG /var/www/drupal /var/cache/yunohost/from_file/scripts
120038 DEBUG + update-alternatives --set php /usr/bin/php7.3
120038 DEBUG update-alternatives: using /usr/bin/php7.3 to provide /usr/bin/php (php) in manual mode
120039 DEBUG + sudo -u drupal env PATH=/var/www/drupal/vendor/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin drush @drupal.prod state:set system.maintenance_mode 1 --input-format=integer
121844 DEBUG + sudo -u drupal env PATH=/var/www/drupal/vendor/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin php7.3 composer.phar update drupal/core webflo/drupal-core-require-dev --with-dependencies
122146 WARNING > DrupalProject\composer\ScriptHandler::checkComposerVersion
122146 WARNING Loading composer repositories with package information
123648 WARNING Updating dependencies (including require-dev)
137881 WARNING Package operations: 48 installs, 0 updates, 0 removals
138182 WARNING   - Installing symfony/browser-kit (v3.4.38): Downloading (0%)GitHub API limit (0 calls/hr) is exhausted, could not fetch https://api.github.com/repos/symfony/browser-kit/zipball/ede0c5fa204586e3fa51053fbea9cea8a5a3ee8b. Create a GitHub OAuth token to go over the API rate limit. You can also wait until ? for the rate limit to reset.
138183 WARNING
138183 WARNING Head to https://github.com/settings/tokens/new?scopes=repo&description=Composer+on+yh01.yh.yalh.net+2020-03-25+1049
138184 WARNING to retrieve a token. It will be stored in "/var/www/drupal/.composer/auth.json" for future use by Composer.
```

And the upgrade is waiting for a validation for the token.

As far as I've seen, to solve that issue there is two ways:
1. adding a token like describe at https://github.com/composer/composer/issues/3542 or https://gist.github.com/jeffersonmartin/d0d4a8dfec90d224d14f250b36c74d2f
2. not exceding the rate limit allows for up to 60 requests per hour for unauthenticated requests => that's why there is a sleep 3600=1 hour ^^

There is no issue for that at https://github.com/YunoHost/issues/issues?q=Rate+limite.
If that workaround works I will create one issue

Reply:
Now I have a package check that goes level 7 => https://ci-apps-dev.yunohost.org/jenkins/job/pixelfed_ynh%20PR101%20(yalh76)/8/console

I will just remove the `sleep 3600` and see what happens

Reply:
With `sleep 3600` removed, now it goes well... The only difference: package_check on ci-apps-dev has been rebuilt by Maniack after a crash...
And it goes well for drupal too...
Weird... but it's now ok